api/database/seeders/BigSeederPoolCandidateUser.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Enums\AssessmentResultType;
+use App\Enums\AssessmentStepType;
+use App\Enums\PoolSkillType;
+use App\Models\AssessmentResult;
+use App\Models\AssessmentStep;
+use App\Models\Community;
+use App\Models\Pool;
+use App\Models\PoolCandidate;
+use App\Models\PoolSkill;
+use App\Models\User;
+use Illuminate\Database\Seeder;
+
+class BigSeederPoolCandidateUser extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * This seeds a lot of data
+     * Run this AFTER core data has been seeded, this will not seed platform data
+     *
+     * User/PoolCandidate/Pool/AssessmentResult
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $input = $this->command->ask('Please enter how many times to loop');
+        $limit = intval($input);
+        $limit = (is_int($limit) && $limit > 0) ? $limit : 1;
+
+        // constant values for reuse and setup
+        $digitalCommunityId = Community::select('id')->where('key', 'digital')->sole()->id;
+        $atipCommunityId = Community::select('id')->where('key', 'atip')->sole()->id;
+        $financeCommunityId = Community::select('id')->where('key', 'finance')->sole()->id;
+
+        $pools = Pool::factory()
+            ->count(2)
+            ->withPoolSkills(4, 4)
+            ->withQuestions(2, 2)
+            ->published()
+            ->withAssessments(3)
+            ->create();
+        $poolIds = $pools->pluck('id')->toArray();
+
+        // User - non-government
+        for ($i = 0; $i < $limit; $i++) {
+            User::factory()
+                ->asApplicant()
+                ->withSkillsAndExperiences()
+                ->afterCreating(function (User $user) use ($poolIds) {
+                    $this->applyToPools($user, $poolIds);
+                })
+                ->count(100)
+                ->create();
+        }
+
+        // User - government
+        for ($i = 0; $i < $limit; $i++) {
+            User::factory()
+                ->asApplicant()
+                ->withSkillsAndExperiences()
+                ->asGovEmployee()
+                ->withEmployeeProfile()
+                ->withCommunityInterests([
```

Comment:
Community interests and assessment results were the most I was looking for .. Thanks!
I don't expect  nomination events more than a single digit at any point of time as they talk about.